rav_kav.py

```diff
@@ -0,0 +1,60 @@
+def add_money(account_balance: float) -> float:
+    """
+        Add money to the current balance and print the status
+        :param account_balance: The user's account balance.
+        :return: The user's account balance.
+    """
+    print("Please enter the amount of money you want to add to the card:",
+          end=" ")
+    money_amount = float(input())
+    account_balance += money_amount
+    print(f"{money_amount}$ were added to your account!")
+    print(f"your account balance is now {account_balance}$")
+    return account_balance
+
+
+def spend_money(account_balance: float, sum_of_expenses: float) -> [float, float]:
```

Comment:
Nice typehints, but you should use `Tuple[float, float]`. This is because you should specify if the two floats are returned in a list or a tuple. This is important because one is mutable and the other is imutable.